virttest/guest_agent.py

```diff
@@ -509,7 +509,10 @@ def shutdown(self, mode=SHUTDOWN_MODE_POWERDOWN):
         if mode in [self.SHUTDOWN_MODE_POWERDOWN, self.SHUTDOWN_MODE_REBOOT,
                     self.SHUTDOWN_MODE_HALT]:
             args = {"mode": mode}
-        self.cmd(cmd=cmd, args=args, success_resp=False)
+        try:
+            self.cmd(cmd=cmd, args=args, success_resp=True)
+        except VAgentCmdError:
```

Comment:
better return the detailed error msg for debugging usage.

Reply:
@lijinlijin  It will return  "The command xxx has been disabled for this instance"  error msg when executing shutdown.  This msg is from VAgentCmdError,   VAgentCmdError depend on qemu monitor.  Do you wish like this "except 'The command xxx has been disabled for this instance'": ?

Reply:
@lijinlijin  In order to tp-qemu can get this error Msg,  raise this error Msg.

Reply:
If you need to check the content of the error message in your test context, here should let `self.cmd()` raise the `VAgentCmdError` directly instead of catching it and returning `False`.

So finally the code would be:
```patch
- self.cmd(cmd=cmd, args=args, success_resp=False)
- return True
+ self.cmd(cmd=cmd, args=args)
```

Reply:
@luckyh  You are right,  cmd() can raise this Error.   I have updated it and re-tested it. 

 (1/2) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.x86_64.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.check_reset_reboot_shutdown_fsfreeze: PASS (124.79 s)
 (2/2) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.x86_64.io-github-autotest-qemu.qemu_guest_agent.isa_serial.check_reset_reboot_shutdown_fsfreeze: PASS (143.99 s)